daemon/src/to_sse_event.rs

```diff
@@ -32,6 +34,35 @@ pub struct Cfd {
     pub state: CfdState,
     pub actions: Vec<CfdAction>,
     pub state_transition_timestamp: u64,
+
+    pub details: CfdDetails,
+}
+
+#[derive(Debug, Clone, Serialize)]
+pub struct CfdDetails {
+    tx_url_list: Vec<TxUrl>,
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc::opt")]
+    payout: Option<Amount>,
+}
+
+#[derive(Debug, Clone, Serialize)]
+pub struct TxUrl {
+    pub label: String,
+    pub url: String,
+}
+
+impl TxUrl {
+    pub fn new(txid: Txid, network: Network, label: String) -> Self {
+        Self {
+            label,
+            url: match network {
+                Network::Bitcoin => format!("https://mempool.space/tx/{}", txid),
```

Comment:
Could use the `Url` type here.

Reply:
That just makes the constructions unnecessarily clunky - we have `expects` there then. I think the `String` is good enough for now, can tackle in a follow up if we think it's important.